rust/core-lib/src/plugins/rpc_types.rs

```diff
@@ -127,10 +127,35 @@ pub struct Span {
 }
 
 #[derive(Serialize, Deserialize, Debug)]
+pub struct ScopeSpan {
+    pub start: usize,
+    pub end: usize,
+    pub scope_id: u32,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Style {
+    pub fg: u32,
+    #[serde(rename = "font")]
+    pub font_style: u8,
```

Comment:
Why isn't this just named `font` and then not renamed? And if it is to clarify that it doesn't really hold a font, wouldn't `font_id` be better (without even a serde rename)?